railties/lib/rails/generators/rails/db/system/change/change_generator.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require "rails/generators/base"
+
+module Rails
+  module Generators
+    module Db
+      module System
+        class ChangeGenerator < Base # :nodoc:
+          include Database
+          include AppName
+
+          class_option :to, required: true,
+            desc: "The DBMS to switch to."
+
+          def self.default_generator_root
+            path = File.expand_path(File.join(base_name, "app"), base_root)
+            path if File.exist?(path)
+          end
+
+          def initialize(*)
+            super
+            if !DATABASES.include?(options[:to])
+              raise Error, "Invalid value for --to option. Supported for preconfiguration are: #{DATABASES.join(", ")}."
```

Comment:
This sentence reads a little weird to me. I think "Supported preconfigurations are" would sound better than "Supported for preconfiguration are"